Configuration/PyReleaseValidation/python/MatrixInjector.py

```diff
@@ -83,7 +83,8 @@ def __init__(self,opt,mode='init',options=''):
         self.speciallabel=''
         if opt.label:
             self.speciallabel= '_'+opt.label
-
+        self.countLongWFName = 0
+        self.longWFName = ''
```

Comment:
make this a list, then the separate "count" is unneeded

Reply:
Fixed, make the list instead.